File: python/iniciante/bee_1038/main.py
input = input('')

code = int(input.split()[0])
qtd = int(input.split()[1])

price = 0
if code == 1:
    price = 4
elif code == 2:
    price = 4.5
elif code == 3:
    price = 5
elif code == 4:
    price = 2
elif code == 5:
    price = 1.5

total = price*qtd
print(f'Total: R$ {total:.2f}')

File: rust/iniciante/bee_1038/src/main.rs
use std::io;

fn read_input() -> (i32, i32) {
    let mut input = String::new();

    io::stdin()
    .read_line(&mut input)
    .expect("");

    let mut input = input.split_whitespace();

    // Idk if this kind of syntax is beauty...
    (
    input.next()
    .unwrap()
    .parse()
    .unwrap(),
    input.next()
    .unwrap()
    .parse()
    .unwrap(),
    )
}

fn main() {
   let (code, quantity) = read_input();

    let price = match code {
        1 => 4.00,
        2 => 4.50,
        3 => 5.00,
        4 => 2.00,
        5 => 1.50,
        _ => 0.0,
    };

    let value = price * f64::from(quantity);
    println!("Total: R$ {:.2}", value);
}
